File: 02/2.rb
def main
	a , b , total = 0 , 1 , 0
	while a <= 4000000 do 
		tmp = a + b
		b = a
		a  = tmp
		if a % 2 == 0 
			total += a
		end
	end
	puts total
end

if __FILE__ == $0
	main
end

File: 02/2.c
#include <stdio.h>

int main ()
{
    long a = 0 , b = 1 , total = 0;
	while (a <= 4000000)
	{
		long tmp = a + b;
		b = a;
		a = tmp;
		if (a % 2 == 0)
			total += a;
	}
	printf("%d\n" , total);
}
